yotiprotoattr/Attribute.proto

```diff
@@ -1,11 +1,11 @@
 syntax = "proto3";
 
-package Yoti.Protobuf.attrpubapi_v3;
+package attrpubapi_v1;
 
 option java_package = "com.yoti.attrpubapi_v3";
```

Comment:
Should you remove it here too?

Reply:
This won't ever be used by Java, since we're going to refer to the main lampkicking repo's proto files, but I guess there's no harm in changing it, I'll update it.